### ISSUE TITLE
[Feature Request] Send the ticket ONLY to the collaborator

### ISSUE THREAD

**Issue by rob-tv** [osTicket/osTicket#5820]
Hello,
I'd like to know if you could please add a feature like sending a ticket only to the collaborator, thus excluding the original ticket sender.
Sometimes this could be useful for example if you need to ask a separate authorization from the collaborator (which could be your boss) prior to execute the ticket request, and you don't want the original sender to see it.

Hope you can fulfill my request,
thanks a lot

Roberto
Italy

**Comment by JediKev** [comment]
@rob-tv

Just a thought, you can create a new ticket from a thread entry with the owner being the collaborator and you can link the tickets so you can easily reference them. This way you have total separate communication to the CC that the original owner can’t see and you have references to both tickets.

Cheers.

**Comment by rob-tv** [comment]
I tried but every time you have to fill some mandatory fields (Ticket source, Request, Issue summary...) and, processing a lot of tickets every day, it's a bit slower and more complicated than just selecting the "Reply to" dropdown and choose an eventual "Collaborators" selection.

Regards

**Comment by Mike-Benoit** [comment]
Yes, this would be a nice feature. And/or a quick way to swap the owner with a collaborator in case where the collaborator "takes over" the ticket, but the original owner wants to still be involved.
